src/butil/containers/flat_map_inl.h

```diff
@@ -273,7 +273,7 @@ FlatMap<_K, _T, _H, _E, _S>::operator=(const FlatMap<_K, _T, _H, _E, _S>& rhs) {
         free(_buckets);
         _nbucket = rhs._nbucket;
         // note: need an extra bucket to let iterator know where buckets end
-        _buckets = (Bucket*)malloc(sizeof(Bucket) * (_nbucket + 1/*note*/));
+        _buckets = new Bucket[sizeof(Bucket) * (_nbucket + 1/*note*/)];
```

Comment:
_buckets is released by free which should be matched with malloc. And even if new[] is used, the "sizeof(Bucket) *" should be removed.